### PR TITLE
Navigation and footer link fixes.

### PR DESCRIPTION
**Pull request by awetherill**

### CODE REVIEW

File: template/fonts.css [hunk 1]
```diff
@@ -4,34 +4,64 @@
...
   src: url('https://www.nd.edu/assets/fonts/GalaxiePolarisCondensed-Heavy.eot');
   src: url('https://www.nd.edu/assets/fonts/GalaxiePolarisCondensed-Heavy.eot?iefix') format('embedded-opentype'), url('https://www.nd.edu/assets/fonts/GalaxiePolarisCondensed-Heavy.woff') format('woff'), url('https://www.nd.edu/assets/fonts/GalaxiePolarisCondensed-Heavy.ttf') format('truetype'), url('https://www.nd.edu/assets/fonts/GalaxiePolarisCondensed-Heavy.svg#svgGPCHeavy') format('svg');
-}
+}*/
 
 @font-face {
+  font-family: GPCHeavy;
+  src: url('https://static.nd.edu/fonts/gp-min/gpc-heavy.woff2');
+  src:url("https://static.nd.edu/fonts/gp-min/gpc-heavy.woff2") format("woff2"),url("https://static.nd.edu/fonts/gp-min/gpc-heavy.woff") format("woff"),url("https://static.nd.edu/fonts/gp-min/gpc-heavy.ttf") format("truetype")
+}
```
Comment (mlkirkpatrick) [inline template/fonts.css]: These links give an Access Denied.

File: template/fonts.css [hunk 2]
```diff
@@ -40,20 +70,39 @@
...
   font-family: GPHeavy;
   src: url('https://www.nd.edu/assets/fonts/GalaxiePolaris-Heavy.eot');
   src: url('https://www.nd.edu/assets/fonts/GalaxiePolaris-Heavy.eot?iefix') format('embedded-opentype'), url('https://www.nd.edu/assets/fonts/GalaxiePolaris-Heavy.woff') format('woff'), url('https://www.nd.edu/assets/fonts/GalaxiePolaris-Heavy.ttf') format('truetype'), url('https://www.nd.edu/assets/fonts/GalaxiePolaris-Heavy.svg#svgGPCHeavy') format('svg');
+}*/
+
+@font-face {
+  font-family: GPHeavy;
+  src: url('https://library.nd.edu/static/fonts/gp-heavy.woff');
+  src:url("https://library.nd.edu/static/fonts/gp-heavy.woff") format("woff2"),url("https://library.nd.edu/static/fonts/gp-heavy.woff") format("woff"),url("https://library.nd.edu/static/fonts/gp-heavy.ttf") format("truetype")
 }
```
Comment (mlkirkpatrick) [inline template/fonts.css]: 404 Not Found on these.

File: template/fonts.css (hunk 1 above)
Comment (awetherill) [reply template/fonts.css]: Ah, we don't actually use that font anyway so I just commented them out.

File: template/fonts.css (hunk 2 above)
Comment (awetherill) [reply template/fonts.css]: Moved these to a place that should work now.

**Comment by awetherill** [comment]
.

**Pull request merged**